acvm-repo/acvm_js/src/execute.rs

```diff
@@ -314,12 +314,12 @@ impl<'a, B: BlackBoxFunctionSolver<FieldElement>> ProgramExecutor<'a, B> {
                     }
                     ACVMStatus::RequiresAcirCall(call_info) => {
                         let acir_to_call = &self.functions[call_info.id.as_usize()];
-                        let initial_witness = call_info.initial_witness;
+                        let initial_witness = &call_info.initial_witness;
```

Comment:
```suggestion
                        let initial_witness = call_info.initial_witness;
```
why do we borrow here?